nestkernel/conn_builder.cpp

```diff
@@ -114,6 +116,19 @@ nest::ConnBuilder::ConnBuilder( Network& net,
       : ConnParameter::create( ( *syn_defaults )[ names::delay ], net_.get_num_threads() );
   }
   register_parameters_requiring_skipping_( *delay_ );
+  // Structural plasticity parameters
+  // Check if both pre and post synaptic element are provided
+  if ( syn_spec->known( names::pre_synaptic_element )
+    && syn_spec->known( names::post_synaptic_element ) )
```

Comment:
I agree, I was more focused on not disturbing people who doesn't use structural plasticity but we can assume that someone specifing only one synatic_element is doing something wrong.